.envrc

```diff
@@ -0,0 +1,7 @@
+if [[ $(type -t use_flake) != function ]]; then
+  echo "ERROR: use_flake function missing."
+  echo "Please update direnv to v2.30.0 or later."
+  exit 1
+fi
+
```

Comment:
Since we are configuring a new tool for the project, should we add a description on the README?